src/server.test.ts

```diff
@@ -0,0 +1,39 @@
+import fastify from 'fastify';
+import { registerRoutes } from './routes';
+import { joinNotNull } from './shared/utils';
+
+describe('Chaos Stream Proxy server', () => {
+  const app = fastify();
+
+  // Register routes
+  registerRoutes(app);
+
+  // Adding version header
+  app.addHook('onSend', async (request, reply, payload) => {
```

Comment:
The intention with the unit test is to test the implementation in server.ts. I don't really see that this does that?